src/js/components/DateInput/stories/Format.js

```diff
@@ -12,9 +13,30 @@ export const Format = () => {
   };
   return (
     <Box align="center" pad="large">
-      <Box width="medium">
+      <Box width="medium" margin={{ vertical: 'small' }}>
         <DateInput format="m/d/yy" value={value} onChange={onChange} />
       </Box>
+      <Box width="medium" margin={{ vertical: 'small' }}>
+        <DateInput format="m/d/yy" value={value} reverse onChange={onChange} />
+      </Box>
+      <Box width="medium" margin={{ vertical: 'small' }}>
+        <DateInput
+          format="m/d/yy"
+          value={value}
+          icon={<ClockIcon />}
+          inputProps={{ reverse: true }}
```

Comment:
I think we should encourage using the `reverse` prop over setting reverse through input props. Can we change the story examples to use `reverse`?

Reply:
Yes, that makes sense